### PR TITLE
Cu 862j2kjcg start aplikacji od ekranu login form component marcin h

### PR DESCRIPTION
**Pull request by Marcin-H**
- added form login and validationn
- saving token to localstorage
- private routes
- logout
- few improvements

### CODE REVIEW

**Comment by s20146** [comment]
Task linked: [CU-862j2kjcg Start aplikacji od ekranu LoginFormComponent](https://app.clickup.com/t/862j2kjcg)

File: React/src/components/TwoButtons/TwoButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <Grid item>
+        <Button color="secondary" variant="text">
+          {secondBtnValue}
+        </Button>
+      </Grid>
+    </Grid>
+  );
+};
+
+export default TwoButtons;
```
Comment (kurylek) [inline React/src/components/TwoButtons/TwoButtons.tsx]: Do we use this component?

File: React/src/api/Authentication.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { makeServiceCall } from './utils';
+
+type LogInPayloadType = {
+  username: string;
+  password: string;
+};
+
+export const logIn = (data: LogInPayloadType) => {
+  console.log({ data });
```
Comment (kurylek) [inline React/src/api/Authentication.ts:9]: Still needed?

File: React/src/api/utils.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import axios, { Method } from 'axios';
+
+interface PayloadProps {
+  body?: undefined | Record<string, unknown> | unknown;
+  queryStringParams?: undefined | Record<string, unknown>;
+}
+
+export const makeServiceCall = async (url: string, httpMethod: Method, payload: PayloadProps) => {
+  console.log({ payload });
```
Comment (kurylek) [inline React/src/api/utils.ts:9]: Still needed?

File: React/src/components/LoginForm/LoginForm.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+      password: '',
+    },
+    validationSchema: validationSchema,
+    onSubmit: (values) => {
+      return handleSubmit(values.email, values.password);
+    },
+  });
+
+  const handleSubmit = async (email: string, password: string) => {
+    console.log({
```
Comment (kurylek) [inline React/src/components/LoginForm/LoginForm.tsx:56]: Still needed?

**Review by kurylek — APPROVED** [review]

**Pull request merged**